Formula/aliyun-cli.rb

```diff
@@ -0,0 +1,29 @@
+class AliyunCli < Formula
+  desc "Universal Command-line Interface for Alibaba Cloud"
+  homepage "https://github.com/aliyun/aliyun-cli"
+  url "https://github.com/aliyun/aliyun-cli/archive/v3.0.5.tar.gz"
+  sha256 "fa9a9e736f3a02c454695b75e0dd1e7c140fba63d114537d24babd0b6930f99a"
+
+  head do
+    url "https://github.com/aliyun/aliyun-cli.git"
+  end
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dir = buildpath/"src/github.com/aliyun/aliyun-cli"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+
+    cd dir do
+      system "make", "clean", "deps", "build"
+      bin.install "out/aliyun"
+    end
+  end
+
+  test do
+    system "#{bin}/aliyun"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
done

Reply:
I hope the app can do more than print a version number :) so we should test something more serious than this

Reply:
Unfortunately this would involve generating API credentials and running the `configure` command.

This cli functions very much like the aws cli, as in it sends authenticated requests against an API.

I'm not sure what is meant by ` exercises the some of the functionality of the app`, if the app successfully compiles and the `version` command works, isn't that sufficient?

Reply:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/awscli.rb#L49-L51

Reply:
The `awscli` test should be improved, yes. Other formulas, like `azure-cli`, have more meaningful tests. You could make it output a configuration, or try to perform an operation with invalid credentials and check that it fails.

Reply:
More meaningful test has been added and CI is now passing